.github/workflows/pr_env.yaml

```diff
@@ -0,0 +1,149 @@
+name:  Pull request environment
+
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - labeled
+
+env:
+  MANTAINERS: "[\"cdimonaco\", \"dottorblaster\", \"fabriziosestito\", \"rtorrero\", \"nelsonkopliku\", \"arbulu89\",\"jagabomb\",\"emaksy\",\"jamie-suse\"]"
```

Comment:
Could we use some already predefined group instead of individual names?
We have the `Developers` groups for example.
Or `Admins`, or something like that.
Plain users don't look like the best option